Training/Easy/chuck_norris.rb

```diff
@@ -27,25 +27,24 @@ def split_series(array)
     result
   end
 
-  def norris_parse(array)
-    result = ''
-    array.each do |a|
-      if a[0].zero?
-        result += '00 '
-      else
-        result += '0 '
-      end
-      result += '0' * a[1] + ' '
+  def norris_parse(series_array)
+    parsed = series_array.each_with_object('') do |a, result|
+      result << if a[0].zero?
+                  '00 '
+                else
+                  '0 '
+                end
+      result << '0' * a[1] + ' '
     end
-    result.strip
+    parsed.strip
   end
 
   def start
     binary = @message.each_char.each_with_object('') do |char, b|
```

Comment:
You don't need to chain `each_char` with `each_with_object` - `@message` has a `chars` method